as_arm:
  as_arm_joint_controller:
    type: "position_controllers/JointTrajectoryController"
    joints:
      - base_joint
      - shoulder_joint
      - elbow_joint
      - wrist_flex_joint
      - wrist_rot_joint

    gains:
      base_joint:   {p: 1000.0, i: 0.0, d: 0.1, i_clamp: 0.0}
      shoulder_joint: {p: 1000.0, i: 0.0, d: 0.1, i_clamp: 0.0}
      elbow_joint:  {p: 1000.0, i: 0.0, d: 0.1, i_clamp: 0.0}
      wrist_flex_joint:       {p: 1000.0, i: 0.0, d: 0.1, i_clamp: 0.0}
      wrist_rot_joint:    {p: 1000.0, i: 0.0, d: 0.1, i_clamp: 0.0}


  gripper_controller:
    type: "position_controllers/JointTrajectoryController"
    joints:
      - finger_joint1
      - finger_joint2
    gains:
      finger_joint1:  {p: 50.0, d: 1.0, i: 0.01, i_clamp: 1.0}
      finger_joint2:  {p: 50.0, d: 1.0, i: 0.01, i_clamp: 1.0}